go.cap

```diff
@@ -0,0 +1,10 @@
+github.com/onflow/cadence ()
+
+github.com/davecgh/go-spew/spew (file)
+github.com/onflow/cadence/runtime/interpreter (runtime)
+github.com/onflow/cadence/runtime/parser2 (file)
+github.com/onflow/cadence/runtime/pretty (runtime)
+github.com/opentracing/opentracing-go (network)
+github.com/stretchr/testify/assert (runtime, file, network)
+github.com/zeebo/blake3/internal/consts (file)
```

Comment:
I wish we didn't have to allow `file` to any hash libraries.  In this case blake3/internal/consts appears to just call os.GetEnv() and call cpu.* functions.

Reply:
Yeah, same for the other packages: The parser has a utility to read a file, and the `runtime` usage in the interpreter is just to handle Go errors (Go's `runtime.Error`).